### DISCUSSION TITLE
2.6.5 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#2514] [讨论]
· 主页自定义
　　新增“预设”选项，可以直接选择预设的几种主页（#1368）
　　更新联网主页前，先使用本地缓存显示主页（#40）
　　更新联网主页时，会先检查主页版本，仅在版本变更时下载，以节省流量
· XAML 自定义功能扩展
　　新增全局替换标记 {hint}：加载时会被替换为随机“你知道吗？”提示
　　新增全局替换标记 {cave}：加载时会被替换为随机回声洞内容
　　刷新主页事件：若提供了任意 EventData，则不显示“已刷新主页！”提示
　　MyIconButton 控件：支持执行自定义事件
· 帮助库更新
　　添加<多人游戏常见问题>页面（by @youzi-2333、@WForst-Breeze）
　　编修多个页面（by @youzi-2333、@WForst-Breeze）
　　更换图床链接（by @z0z0r4、@Big-Cake-jpg）
· 细节优化
　　允许按 Ctrl+A 全选 Mod（#2334，@hejiehao）
　　为特定 Java 的识别失败添加详细的错误信息（#2347，@RimacC2-EV）
　　清理游戏垃圾的警告在清理 3 次后就不再显示（#2373，@Anythingteam）
　　增加提示条背景的不透明度（#2452，@JingHai-Lingyun）
　　为快照版提示添加了临时关闭按钮
　　内存优化结束时会提示目前总剩余内存
　　当尝试使用高版本 JRE 时，添加换用 JDK 的提示
· 周期性维护与更新
　　翻新大量“你知道吗？”提示与回声洞，提升易读性
　　更新 Mod 数据库中的新增条目
　　1.20.2+ 下改用数据包版本 17
· 修复了以下 13 个 Bug
　　罕见情况下连续开始自定义下载导致出错（#2283，@Bandysol）
　　在鼠标按下时从带图标按钮移开导致颜色异常（#2293，@ZJH-github34987233）
　　Night Config 导致崩溃的提示有误（#2301，by @zkitefly）
　　启动 OptiFine 1.12 时可能选择错误的 Java（#2328，@niuyuhao0818）
　　当所有音乐都无法播放时继续出现错误的报错（#2366，@Madvikfut）
　　罕见情况下无法开始播放音乐（#2392，@SuoHouDaiShi）
　　某些网络请求没有添加 Header（#2476，@Light-Beacon）
　　不支持自动搜索 Liberica JDK（#2480，@Xie-ZJY）
　　Xbox 文本大小写有误（#2486，@DarnedBuzzard70，by @SALTWOOD）
　　只有一首无法播放的音乐时尝试开始播放导致未知报错
　　不支持自动搜索官启下载的 Java
　　Mod 管理的选中提示在切换页面时没有退出动画
　　自定义主页设置的单选框与其他卡片没有纵向对齐

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.6.5'>2.6.5</a>.</em>

**Comment by Chormore-EDGE** [comment]
信息量巨大
~#40 还能被做真没想到~

**Comment by iPodtouch6557** [comment]
~近几个更新都是在我没带电脑出去的情况下更新的~

**Comment by SuoHouDaiShi** [comment]
我这情况确实罕见（
毕竟都3202年了，为啥还有人要用机械硬盘装游戏呢（特指我

**Reply by iPodtouch6557** [reply]
我除了Minecraft啥都不玩，自然没这问题.jpg
